__tests__/Unit/Components/Tabs/Tab.test.tsx

```diff
@@ -1,6 +1,17 @@
 import Tabs from '@/components/Tabs';
 import { fireEvent, render, screen } from '@testing-library/react';
 import { Tab, TABS } from '@/interfaces/task.type';
+import { COMPLETED, DONE, AVAILABLE, UNASSINGED } from '@/constants/constants';
+
+function changeName(name: string) {
+    if (name === COMPLETED) {
+        return DONE;
+    } else if (name === AVAILABLE) {
+        return UNASSINGED;
+    } else {
+        return name.split('_').join(' ');
+    }
+}
```

Comment:
Creating a utility for this would help us reduce the re-writing for this function. (the function change name is defined and used again in the Tabs component)